torchprime/experimental/torchax_models/utils/parallelism_utils.py

```diff
@@ -0,0 +1,141 @@
+from functools import partial
```

Comment:
In relation to folks' comment above, my suggestion is to write a single `parallelism_utils` using vanilla PyTorch. Both PyTorch/XLA and torchax can run vanilla PyTorch, just with different lowering.

If you have to write some JAX code, you may use `torch_xla.core.xla_builder.call_jax`. `call_jax` supports both PyTorch/XLA and torchax tensors. So both frameworks can run this function.

Reply:
thanks, let me try